hydra-tui/src/Hydra/TUI.hs

```diff
@@ -369,10 +369,6 @@ handleAppEvent s = \case
     s & warn' time ("Invalid input error: " <> toText reason)
   Update TimedServerOutput{time, output = PostTxOnChainFailed{postTxError}} ->
     case postTxError of
-      NotEnoughFuel ->
-        s
-          & warn' time "Not enough Fuel. Please provide more to the internal wallet and try again."
-          & stopPending
```

Comment:
This would have been a good hint to _not_ remove `NotEnoughFuel` as it provides clear instructions to the user (much better than the `InternalWalletError` case below)